src/components/smartSearch/Filter.tsx

```diff
@@ -34,12 +34,12 @@ const Filter = ({ filter, onDelete, onEdit }:FilterProps): JSX.Element => {
                                     id={ `misc.smartSearch.timeFrame.preview.${timeFrame}` }
                                     values={{
                                         afterDate: (
-                                            config?.after
+                                            after?.toISOString().slice(0,10)
```

Comment:
I just saw this which I didn't realize before, which is that we're passing a date **string** to the message formatter. A better way (again because of localization) would be to pass a `Date` and then let the translator/localization framework format it according to locale using `between {after, date} and {before, date}` syntax.

The `{varName, date}` syntax indicates that the `varName` variable is of type `date` and should be localized as such. For more control over formatting, a third parameter can be passed, e.g. `{varName, date, short}`, which will render as _2021-07-26_ in Swedish, but _7/26/21_ in American English.

Reply:
I will put this in for the next PR ! just because it's getting so much already and it's a scoped task that doesn't depend on everything else